### PR TITLE
ADD: Teku validator import API

### PR DESCRIPTION
**Pull request by gbayasgalan**
https://github.com/stereum-dev/ethereum-node/issues/163 
https://github.com/stereum-dev/ethereum-node/issues/90

### CODE REVIEW

File: controls/roles/validator-import-api/tasks/main.yml [hunk 1]
```diff
@@ -80,3 +80,30 @@
     become: yes
     with_items: "{{ validator_keys }}"
   when: consClient == "nimbus"
+
+- block:
+  - name: Teku - Get validator service's api token
+    slurp:
+      src: "/opt/app/services/{{ beacon_service }}/data/validator/key-manager/validator-api-bearer"
```
Comment (stefa2k) [inline controls/roles/validator-import-api/tasks/main.yml:87]: this should be the same as with prysm, because the path can change with user input

**Review by stefa2k — CHANGES_REQUESTED** [review]

**Comment by stefa2k** [comment]
This resolves #163 right?

File: controls/roles/validator-import-api/tasks/main.yml (hunk 1 above)
Comment (gbayasgalan) [reply controls/roles/validator-import-api/tasks/main.yml:87]: can it be like nimbus or lighthouse? 
https://github.com/stereum-dev/ethereum-node/blob/main/controls/roles/validator-import-api/tasks/main.yml#L43

Comment (stefa2k) [reply controls/roles/validator-import-api/tasks/main.yml:87]: sure, if it works :)

Comment (gbayasgalan) [reply controls/roles/validator-import-api/tasks/main.yml]: done!

**Review by stefa2k — APPROVED** [review]
Nice!

**Pull request merged**